### PR TITLE
Update debuild.sh

### PR DESCRIPTION
**Pull request by irahulcse**
### Short description
I have:
- [ ] There is a corresponding issue for this pull request.
- [ ] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

File: scripts/debuild.sh [hunk 1]
```diff
@@ -3,7 +3,9 @@
 # on travis in /home/travis/build/fossasia/meilix
 
 #rm meilix-default-settings_*                                    #removes the older meilix-default-settings packages if exist
-cd meilix-default-settings                                      #cd into the metapackage directory
+cd meilix-default-settings                                 #cd into the metapackage directory 
+cd ..
+sudo rm melix-default-settings                                  #again trying to remove the melix-default settings package if exist from root user
```
Comment (tabesin) [inline scripts/debuild.sh]: You remove it, then you debuild it? I don't get it.

Comment (tabesin) [reply scripts/debuild.sh]: see line 7

Comment (irahulcse) [reply scripts/debuild.sh]: @tabesin Could you pls guide me towards this issue??

**Comment by rpotter12** [comment]
@irahulcse please always open issue before creating PR.

**Comment by irahulcse** [comment]
> @irahulcse please always open issue before creating PR.

Issue was already discussed with the @tabesin but i will take care of it in future

**Comment by tabesin** [comment]
Meilix-default-settings is absolutely essential. No. Don't "remove" it.

Meilix-artwork is now external and the folder could be removed provided you do it in a way that keeps the history.

**Pull request closed without merging**